### ISSUE TITLE
Distribution Matching for Crowd Counting

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#136]
# TL;DR
Existing crowd counting methods  have generally used Gaussian smoothing for correct data represented by dots, but they have shown experimentally/theoretically that this has a negative impact on the generalization performance. They also proposed a loss function based on the optimal transport and SOTA on various datasets.
![image](https://user-images.githubusercontent.com/24524018/95273831-099bb400-087f-11eb-93c0-4db793d659fc.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2009.13077

# Submission Dates(yyyy/mm/dd)
2020/09/28

# Authors and institutions

Boyu Wang, Huidong Liu, Dimitris Samaras, Minh Hoai
- Stony Brook University

# Methods

# Results

# Comments
